### DISCUSSION TITLE
Customise the list of posts

### DISCUSSION THREAD

**Discussion by snicoll** [lxndrblz/anatole#413] [Q&A] [answered]
Thanks for your work on this theme. I am new to Hugo so I hope this makes sense.

I am a backend developer and writing a web site as a way to showcase my articles (blog posts typically) and my presentations. For each presentation, I have something of that form:

```
+++
author = "Stéphane Nicoll"
date = "2015-12-10"
title = "Spring Boot for the web tier"
conference = "Devoxx"
countryCode = 'BE'
redirectUrl = "https://www.youtube.com/watch?v=7cemYpV1fNI"
+++
```

I'd like the "summary" in the main page to show the 🇧🇪 flag (based on the `BE`) country code, as well as the name of the conference (potentially massaged with a link to the conference, based on its name). I'd typically add this where tags are currently displayed in the summary.

Can someone help me get started on this? Thanks!

**Comment by lxndrblz** [comment, marked as answer]
@snicoll You would need to modify this section of the template:

https://github.com/lxndrblz/anatole/blob/82847f76262b2822edc8879a09983e192712eeab/layouts/index.html#L65-L92

Similarly to the date, you would need to add the conference and CountryCode as params, which will then be displayed in the footer of each post on the start page.

Hope that helps.

**Comment by snicoll** [comment]
Thanks, I'll give that a try.
